probe-rs/src/session.rs

```diff
@@ -379,18 +372,18 @@ impl Session {
 
         // Wait for the cores to be halted.
         for core_id in 0..session.cores.len() {
-            let mut core = session.core(core_id)?;
-
-            core.halt(Duration::from_millis(100))?;
+            match session.core(core_id) {
```

Comment:
I'm open to suggestions how we should actually ignore disabled cores. This works, but isn't particularly nice.

Reply:
Imo this is the best way currently to do this, just because a core is not available now doesn't mean it couldn't be later down the line so ignoring it completely would be a bad idea